exo-create/test_helpers/test_helpers.go

```diff
@@ -0,0 +1,26 @@
+package testHelpers
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+)
+
+func EmptyDir(dir string) error {
+	if err := os.RemoveAll(dir); err != nil {
+		return err
+	}
+	return os.Mkdir(dir, os.FileMode(0777))
+}
+
+func ReformatCommand(command, cwd string) []string {
+	return strings.Split(path.Join(cwd, "bin", command), " ")
+}
+
+func ValidateTextContains(haystack, needle string) error {
+	if strings.Contains(haystack, needle) {
+		return nil
+	}
+	return fmt.Errorf("Expected:\n\n%s\n\nto include\n\n%s", haystack, needle)
```

Comment:
The template can be defined outside the funcs so we can more easily see the formatting
```go
var validateTextContainsErrorTemplate = `
Expected:

%s

to include

%s
`
```